src/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -8,6 +8,7 @@ import { TransactionDetailsSummaryItemFragmentContainer as TransactionDetailsSum
 import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
 import { Dialog, injectDialog } from "Apps/Order/Dialogs"
 import { trackPageViewWrapper } from "Apps/Order/Utils/trackPageViewWrapper"
+import { ActionType, Flow, track } from "Artsy/Analytics"
```

Comment:
Conventionally this is imported as `import * as Schema`

Reply:
:+1: I changed this

Although I'm no longer a fan of `import * as X` since vscode got auto-import but it only works with named imports :cry: